src/includes_normalize-win32.cc

```diff
@@ -26,6 +26,22 @@
 
 namespace {
 
+bool InternalGetFullPathName(LPCTSTR lpFileName, DWORD nBufferLength,
```

Comment:
nit: I get that this is supposed to match GetFullPathName's declaration, but no polish notation please :-) Also, every caller passes NULL for lpFilePart, so I'd omit this parameter here.